### PR TITLE
Standalone Sensor Mode - Send error & events to special events file

### PR DESCRIPTION
**Pull request by iximiuz**

### CODE REVIEW

File: pkg/app/sensor/app.go [hunk 1]
```diff
@@ -27,13 +28,15 @@ import (
...
 	"github.com/docker-slim/docker-slim/pkg/util/errutil"
+	"github.com/docker-slim/docker-slim/pkg/util/fsutil"
 	"golang.org/x/sys/unix"
 
 	log "github.com/sirupsen/logrus"
 )
 
 const (
 	defaultArtifactDirName = "/opt/dockerslim/artifacts"
+	defaultEventsFileName  = "/opt/dockerslim/events.json"
```
Comment (x-private-one) [inline pkg/app/sensor/app.go:39]: is there a specific reason why it's not `/opt/dockerslim/artifacts/events.json`?

Comment (kcq) [inline pkg/app/sensor/app.go]: is there a specific reason why it's not `/opt/dockerslim/artifacts/events.json`?

File: pkg/app/sensor/app.go [hunk 2]
```diff
@@ -489,8 +526,14 @@ func readCommandFile() (command.StartMonitor, error) {
...
 	if err := json.Unmarshal([]byte(jsonCmd), &cmd); err != nil {
-		return cmd, err
+		return cmd, fmt.Errorf("could not decode command: %w", err)
 	}
 
 	return cmd, nil
 }
+
+func writeJSON(v interface{}, w io.Writer) {
+	if err := json.NewEncoder(w).Encode(v); err != nil {
```
Comment (kcq) [inline pkg/app/sensor/app.go]: need to disable the default html escaping for the json encoder object

File: pkg/app/sensor/app.go [hunk 3]
```diff
@@ -381,6 +388,36 @@ func startMonitor(
...
+		case <-ctx.Done():
+			log.Debug("sensor: event collector - done...")
+			return
+		case evt := <-eventChan:
+			log.WithField("event", evt).Info("sensor: event collector - dumping event")
+			writeJSON(evt, out)
+		case err := <-errorChan:
+			log.WithError(err).Warn("sensor: event collector - dumping error")
+			writeJSON(struct {
+				Error string `json:"error"`
```
Comment (kcq) [inline pkg/app/sensor/app.go]: we need to use the event structure we have for the other events: `event.Message{Name: event.Error, Data: err.Error()}`

File: pkg/app/sensor/app.go (hunk 1 above)
Comment (iximiuz) [reply pkg/app/sensor/app.go]: Not really :) Good idea to move it!

File: pkg/app/sensor/app.go (hunk 2 above)
Comment (iximiuz) [reply pkg/app/sensor/app.go]: Done!

File: pkg/app/sensor/app.go (hunk 3 above)
Comment (iximiuz) [reply pkg/app/sensor/app.go]: Done!

**Pull request closed without merging**